EIPS/eip-4955.md

```diff
@@ -0,0 +1,85 @@
+---
+eip: 4955
+title: Non-Fungible Token Metadata Standard Extension
+description:  A metadata standard extension schema for non-fungible tokens
```

Comment:
We don't really want the description to be a more verbose title that provides no more information, so might I suggest:

```suggestion
description:  Add a new field to NFT metadata to store vendor specific data
```